.github/workflows/test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
       - name: Test NetworkX
         run: |
-          pytest -n auto --durations=10 --pyargs networkx
+          pytest -n auto --durations=10 --pyargs --mpl networkx
```

Comment:
We should remove it from here as well as the `base` tests don't include the default dependencies (i.e. matplotlib). Of course `pytest.importorskip` should catch all this for us, but we can explicitly